kitsune/log_settings.py

```diff
@@ -84,3 +84,4 @@
     )
 
 dictConfig(config)
+logging.captureWarnings(True)
```

Comment:
Anything with `DEBUG=True` gets most logging sent to stdout, including warnings. This should cover the runserver case. I don't have a great way to generate warnings in local dev right now, but I assume this would look slightly different but still have the same info.

Anything with `DEBUG=False` gets most logging sent to syslog, where webops can slice and dice it as they see fit, and maybe someday we'll be able to look at it. This should cover the celery and wsgi cases.